frontend/src/pages/selectcategory/selectcategory.page.tsx

```diff
@@ -1,6 +1,49 @@
 import { Card, CardHeader, CardTitle } from "@/components/ui/card";
+import { ROUTES } from "@/routes";
 import { Link } from "react-router-dom";
 
+const categoryList = [
+  {
+    id: 1,
+    name: 'Fun Quizzes',
+    image: 'https://blog.woobox.com/wp-content/uploads/8-Best-Quiz-Ideas-to-Entertain-Inform-Your-Audience-v1.png',
```

Comment:
**suggestion (non-blocking):** Please add placeholder images.
Please use this URL (https://placehold.co/) to generate placeholder images.

Reply:
Changes to placeholder image from thirdparty image url